lib/rocket_chat/messages/room.rb

```diff
@@ -73,6 +162,25 @@ def rename(room_id, new_name)
         )['success']
       end
 
+      #
+      # *.set* REST API
+      # @param [String] room_id Rocket.Chat room id
+      # @param [String] new_name New room name
+      # @param [Hash] setting Single key-value
+      # @return [Boolean]
+      # @raise [ArgumentError, HTTPError, StatusError]
+      #
+      def set_attr(room_id: nil, name: nil, **setting)
+        attribute, value = setting.first
+        validate_attribute(attribute)
+        session.request_json(
+          self.class.api_path(Util.camelize("set_#{attribute}")),
+          method: :post,
+          body: room_params(room_id, name)
```

Comment:
Not related to this change directly, but would also make sense to raise an exception in `room_params` if neither id or name are set